velox/exec/OperatorTraceWriter.cpp

```diff
@@ -96,4 +100,47 @@ void OperatorTraceWriter::writeSummary() const {
   file->close();
 }
 
+OperatorTraceSplitWriter::OperatorTraceSplitWriter(std::string traceDir)
+    : traceDir_(std::move(traceDir)),
+      fs_(filesystems::getFileSystem(traceDir_, nullptr)) {
+  VELOX_CHECK_NOT_NULL(fs_);
+  splitFile_ = fs_->openFileForWrite(getOpTraceSplitFilePath(traceDir_));
+  VELOX_CHECK_NOT_NULL(splitFile_);
+}
+
+void OperatorTraceSplitWriter::write(const exec::Split& split) const {
+  VELOX_CHECK(!split.hasGroup(), "Do not support grouped execution");
```

Comment:
I guess we could support group later once we have driver id mapping in trace node? Leave a TODO here? thanks!